mozilla_aws_cli/static/index.js

```diff
@@ -129,17 +151,18 @@ const pollState = setInterval(async () => {
         setMessage("Another federation session has been detected. Shutting down.");
         clearInternal(pollState);
     } else if (remoteState.state === "error") {
-        setMessage(remoteState.value.message);
+        setMessage("Encountered error : " + remoteState.value);
```

Comment:
Use template literals here.  :)

Reply:
Done in 4c19b5d43a469e4c2fccef8538be1c2f00c7cb31